sponge.go

```diff
@@ -24,13 +27,38 @@ import "sync"
        - economist
    - filter Hacker News if no url
 */
-var itemsToFetch = 10
-var outputLocation = "/tmp/sponge_out.txt"
+
+var defaultOutput = filepath.Join(os.TempDir(), "sponge_out.txt")
```

Comment:
I think this is fine and that's probably mirrored in this passing the compiler's dependency graph.

However!  There's something cool in golang that might be useful to know even if not strictly necessary here:

```
func init() {
    doSomeStuff();
    doSomeMoreStuff();
}

func init() {
    andSomethingElseThatsGuaranteedToRunAfterTheStuffAboveIt();
}
```

I like to do stuff like create stuff at the top level that are dependent on other top level variables using this pattern instead.  So I'd declare var outputLocation *string and then set it in the init.  It would keep polution out of the top level (not really super important since it's private polution but still good hygeine) and be really, really explicit about the fact that there is a calculation happening in order to initialize the variable.

Also, I keep the func init *really really* close to the variable declaration.  Like, the line after it kinda close.